doc/content/integrations/webhooks/webhook-templates/_index.md

```diff
@@ -2,6 +2,7 @@
 title: "Webhook Templates"
 description: ""
 summary: Webhook templates define a webhook integration that is not created (yet). Templates allows for using common values for many webhooks, such as a common base paths.
+weight: 1
```

Comment:
why?

Reply:
I thought it made more sense to have the generic documentation listed before the platforms-specific one, but I can change this if you disagree

Reply:
OK but that it would merely serve as an introduction?

Reply:
When I put this weight like this, we have an Overview section where the whole generic process of creating a webhook is explained, then this section Webhook Templates, then the Cloud Integrations section. I don't see the Webhook Templates section being an introduction here, am I missing something here?
![Screenshot from 2020-06-12 13-08-27](https://user-images.githubusercontent.com/65215579/84496782-d9b41980-acad-11ea-8f8e-76a792d52a69.png)

Reply:
I see now, thanks, this looks good!

Sorry for the confusion